### PR TITLE
added JSON file containing messages

### PR DESCRIPTION
**Pull request by castle-c**

### CODE REVIEW

File: messages.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{"messages":[
+    {"message": "I stepped on a Corn Flake, now I'm a Cereal Killer"},
+    {"message": "On a scale from one to ten what is your favourite colour of the alphabet.."},
+    {"message": "Baby turn around, And let me see that sexy body go, Bump bump bump"},
+    {"message": "Remember that failure is an event, not a person."},
+    {"message": "We don’t see things as they are, we see them as we are."},
+]}
```
Comment (aoswalt) [inline messages.json]: Should have eof newline. Could do standard format.

**Comment by carter-harris** [comment]
👍

**Comment by aoswalt** [comment]
:+1:

**Comment by MelanieRBaker** [comment]
👍

**Pull request merged**